wgsl/index.bs

```diff
@@ -2467,7 +2467,7 @@ to WGSL.
     struct small_stride {
       a: array<f32,8> // stride 4
     }
-    @group(0) @binding(0) var<uniform> invalid: small_stride; // Invalid
+    @group(0) @binding(0) var<uniform> invalid: small_stride; // Invalid, stride must be a multiple of 16
```

Comment:
This probably wraps around badly. Please put the comment before the var declaration.

Reply:
done